src/ert/libres_facade.py

```diff
@@ -52,65 +44,30 @@ class LibresFacade:
 
     def __init__(self, ert_config: ErtConfig, _: Any = None):
         self.config = ert_config
-        self.update_snapshots: dict[str, SmootherSnapshot] = {}
-        self.update_configuration = None
-
-    def smoother_update(
-        self,
-        prior_storage: Ensemble,
-        posterior_storage: Ensemble,
-        run_id: str,
-        observations: Iterable[str],
-        parameters: Iterable[str],
-        progress_callback: Callable[[AnalysisEvent], None] | None = None,
-        global_std_scaling: float = 1.0,
-        rng: np.random.Generator | None = None,
-        misfit_process: bool = False,
-    ) -> SmootherSnapshot:
-        if rng is None:
-            rng = np.random.default_rng()
-        update_snapshot = smoother_update(
-            prior_storage,
-            posterior_storage,
-            observations,
-            parameters,
-            self.config.analysis_config.observation_settings,
-            self.config.analysis_config.es_module,
-            rng,
-            progress_callback,
-            global_std_scaling,
-        )
-        self.update_snapshots[run_id] = update_snapshot
-        return update_snapshot
 
     @property
     def enspath(self) -> str:
+        # Note: only used in tests, ert-internal-examples
+        # semeio ahmanalysis tests, 2do remove?
         return self.config.ens_path
 
-    @property
-    def user_config_file(self) -> str | None:
-        return self.config.user_config_file
-
-    def get_field_parameters(self) -> list[str]:
-        return [
-            val.name
-            for val in self.config.ensemble_config.parameter_configuration
-            if isinstance(val, Field)
-        ]
-
     def get_ensemble_size(self) -> int:
+        # Note: ErtMainWindow on init
+        # 2do get it somehow else / expose through endpoint
         return self.config.runpath_config.num_realizations
```

Comment:
Should be safe to remove ensemble_size from ExperimentPanel.init and get it from config